api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -154,6 +154,13 @@ public void testGetAllExtensionsAndIsSortedByOrder() throws Exception {
 
     }
 
+	@Test
+	public void testGetAllExtensionsWithNoExtensionPointId() throws Exception {
```

Comment:
you inroduced getAllExtensions() but i am seeing the name of the method you are testing is different!!

Reply:
its the same